docs/DeveloperGuide.md

```diff
@@ -467,7 +439,7 @@ During the SetTablesCommand execution:
 4. There is no add/remove/edit table or table list
     * Again, this is because reservations are linked to the tables themselves and if the user alters the list of tables, the reservations will not be in sync.
 
-#### Future Implementations
+**Future Implementations:**
```

Comment:
Actually do we need this part?

Reply:
MB. It was supposed to be called alternative considered as per the textbook instructions. Thanks for spotting that !

Reply:
Hey no problem!